COPYRIGHT.txt

```diff
@@ -139,6 +139,11 @@ Comment: Etc2Comp
 Copyright: 2015, Etc2Comp Authors
 License: Apache-2.0
 
+Files: ./thirdparty/etcpak/
+Comment: etcpak
+Copyright: 2013-2021, Bartosz Taudul <wolf@nereid.pl>
```

Comment:
We don't include emails for other copyright statements in that file so it's probably best to do the same here:
```suggestion
Copyright: 2013-2021, Bartosz Taudul
```